src/Cairo.jl

```diff
@@ -30,14 +30,23 @@ end
 
 include("constants.jl")
 
+# check library version, error message if too low
+cairo_version = ccall((:cairo_version,Cairo._jl_libcairo),Int32,())
+
+if cairo_version < 11200
+    error("cairo version " * string(cairo_version) * ", less then 1.12.00")
```

Comment:
first part: I tend to agree, however i don't see how BinDeps could check cairo_version itself?
part two: ... i remember a lengthy discussion about conditional exports ...

It's not nice, but the best i could come up with.